release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -3123,4 +3123,44 @@ $ oc adm release info 4.11.21 --pullspecs
 [id="ocp-4-11-21-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-11-22"]
+=== RHBA-2023:0027 - {product-title} 4.11.22 bug fix update
+
+Issued: 2023-01-09
+
+{product-title} release 4.11.22 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2022:9107[RHBA-2023:0027] advisory. There are no RPM packages for this release.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.11.22 --pullspecs
+----
+
+[id="ocp-4-11-22-bug-fixes"]
+==== Bug fixes
+
+* The {product-title} {product-version} release updates the `install-config.yaml` file to now list the `me-west1`, (Tel Aviv, Israel) region. After you install the {product-title} by running the `openshift-install` binary, you can select the `me-west1` region for your chosen cluster. (link:https://issues.redhat.com/browse/OCPBUGS-4720[*OCPBUGS-4720*])
+
+* Previously, some object storage instances responded with a `204 No Content` error message when there is was content to display. The {rh-openstack-first} SDK used in {product-title} does not handle 204s correctly. With this update, the installation program works around the issue when there are zero objects to list in a Swift container. (link:https://issues.redhat.com/browse/OCPBUGS-5078[*OCPBUGS-5078*])
```

Comment:
when there is was content to display  >> when there is **no** content to display